### ISSUE TITLE
DST bug

### ISSUE THREAD

**Issue by keanulee** [keanulee/ClassScheduleToICS#4]
Currently, class times are off by an hour during DST. This is because the output ical file uses UTC start and end times. Instead, it should use timezoned times:

```
BEGIN:VEVENT
DTSTART;TZID=America/Toronto:20150112T100000
DTEND;TZID=America/Toronto:20150112T112000
RRULE:FREQ=WEEKLY;UNTIL=20150406T140000Z;BYDAY=MO,WE
...
END:VEVENT
```

This excerpt is from an export from Google Calendar where the class times are correct. Note that the `UNTIL` field still uses UTC (10:00 EDT == 14:00 UTC), so some conversion might need to be done here if it does not support `TZID` (need to research/test/verify).

**Comment by keanulee** [comment]
@bhamodi This might be the fix you're looking for.

**Comment by keanulee** [comment]
The latter issue with `TZID` in `UNTIL` can be ignored if we just assume that it's always _not_ DST so that the `UNTIL` field can be zero or one hour after the last start time. We can do this since a class won't repeat an hour later on the same day. For the above example, the following should produce the same event:

```
...
RRULE:FREQ=WEEKLY;UNTIL=20150406T150000Z;BYDAY=MO,WE
...
```

**Comment by bhamodi** [comment]
"If observance is known to have an effective end date, the `UNTIL` recurrence rule parameter MUST be used to specify the last valid onset of this observance (i.e., the UNTIL date-time will be equal to the last instance generated by the recurrence pattern). It MUST be specified in UTC time."

I am able to make all the other parameters time zone specific, except `RRULE UNTIL` it appears.

**Comment by keanulee** [comment]
Fixed in #5
